src/main/java/picard/illumina/CollectIlluminaBasecallingMetrics.java

```diff
@@ -92,29 +105,31 @@
             "<a href='http://broadinstitute.github.io/picard/picard-metric-definitions.html#IlluminaBasecallingMetrics'>definitions</a> " +
             "for a complete description of the metrics produced by this tool.  </p>" +
 
-            "<hr />"
-    ;
+            "<hr />";
     //Command Line Arguments
 
-    @Option(doc="The Illumina basecalls output directory from which data are read", shortName="B")
+    @Option(doc = "The Illumina basecalls output directory from which data are read", shortName = "B")
     public File BASECALLS_DIR;
 
     @Option(doc = "The barcodes directory with _barcode.txt files (generated by ExtractIlluminaBarcodes). If not set, use BASECALLS_DIR. ", shortName = "BCD", optional = true)
     public File BARCODES_DIR;
 
-    @Option(doc="The lane whose data will be read", shortName = StandardOptionDefinitions.LANE_SHORT_NAME)
+    @Option(doc = "The lane whose data will be read", shortName = StandardOptionDefinitions.LANE_SHORT_NAME)
     public Integer LANE;
 
     // TODO: No longer optional after old workflows are through
-    @Option(doc="The file containing barcodes to expect from the run - barcodeData.#",shortName=StandardOptionDefinitions.INPUT_SHORT_NAME, optional = true)
+    @Option(doc = "The file containing barcodes to expect from the run - barcodeData.#", shortName = StandardOptionDefinitions.INPUT_SHORT_NAME, optional = true)
     public File INPUT;
 
-    @Option(doc=ReadStructure.PARAMETER_DOC, shortName="RS")
+    @Option(doc = ReadStructure.PARAMETER_DOC, shortName = "RS")
     public String READ_STRUCTURE;
 
-    @Option(doc="The file to which the collected metrics are written", shortName= StandardOptionDefinitions.OUTPUT_SHORT_NAME, optional = true)
+    @Option(doc = "The file to which the collected metrics are written", shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, optional = true)
     public File OUTPUT;
 
+    @Option(doc = "Use the new converter", optional = true)
```

Comment:
When does this need be used?  It would be nice not to have to know the type of sequencer data you have and it "just works".  I am not sure that's the case anymore.